Makefile

```diff
@@ -91,7 +91,7 @@ clean: stop
 
 distclean: clean
 # KJM: Removed cleanup of stack-logs since this target will be part of normal update workflow
-	rm -rf keys opt-auth opt-apigateway dpx-apigateway*.env dpx-vplugin-mgr*.env certs-selfsigned certs-letsencrypt api_key catalogic-dpx-ms.id certbot plugins rest-db
+	docker swarm leave --force && docker swarm init && rm -rf keys opt-auth opt-apigateway dpx-apigateway*.env dpx-vplugin-mgr*.env certs-selfsigned certs-letsencrypt api_key catalogic-dpx-ms.id certbot plugins rest-db
```

Comment:
Would a complete re-initialization of the single-node swarm cause any issues here? Just concerned with `distclean` running a blanket `leave --force` after a swarm is already started and working.